build/build.proj

```diff
@@ -121,7 +151,7 @@
     Build, Pack, Core Tests, Unit tests for VS 15
     ============================================================
   -->
-  <Target Name="RunVS15"  DependsOnTargets="BuildVS15;Pack;CoreUnitTests;UnitTestsVS15">
+  <Target Name="RunVS15"  DependsOnTargets="BuildVS15;Pack;CoreUnitTests;UnitTestsVS15;ApexTestsVS15">
```

Comment:
Good point, I'll remove this for now. I think this will need to run directly on a different machine as part of a new E2E run.